wp-content/plugins/hurumap/src/CardBlock/Save.js

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import propTypes from '../propTypes';
-import config from '../config';
 
 function Save({ attributes }) {
   return (
-    <iframe
-      frameBorder={0}
-      width={attributes.cardWidth}
+    <div
       id={`hurumap-card-${attributes.id}`}
-      style={{ marginLeft: 10, marginBottom: 10 }}
-      title={`${attributes.postType}-${attributes.postId}`}
-      src={`${config.WP_BACKEND_URL}/card/${attributes.postType}/${attributes.postId}?width=${attributes.cardWidth}&id=${attributes.id}`}
+      style={{
+        marginLeft: 10,
+        marginBottom: 10,
+        width: attributes.cardWidth
+      }}
```

Comment:
`class` won't work? Harder to override styles defined using `style`

Reply:
I'll resolve this in another PR